api/controllers/cartController.js

```diff
@@ -31,19 +31,18 @@ const getCartList = async (req, res) => {
   }
 };
 
-const updateCartItem = async (req, res) => {
+const deleteCart = async (req, res) => {
```

Comment:
개별 삭제 기능도 있는 것 같은데, 그렇다면 구별을 위해 다음과 같이 함수 이름을 바꿔주면 좋을 것 같습니다.
```suggestion
const deleteAllCart = async (req, res) => {
```